diff --git a/kernel/runtime/boot_event.c b/kernel/runtime/boot_event.c
--- a/kernel/runtime/boot_event.c
+++ b/kernel/runtime/boot_event.c
@@ -15,6 +15,4 @@
 bool ksu_boot_completed __read_mostly = false;
 extern struct static_key_true ksu_is_input_hook_enabled;

-extern void ksu_avc_spoof_late_init(void);
-
 void on_post_fs_data(void)
@@ -70,8 +68,7 @@
 void on_boot_completed(void)
 {
     ksu_boot_completed = true;
     pr_info("on_boot_completed!\n");
     track_throne(true);
     ksu_selinux_hide_drop_backup_if_unused();
-    ksu_avc_spoof_late_init();
 }
