diff --git a/kernel/selinux/rules.c b/kernel/selinux/rules.c
index 9c6b8674..e62d0687 100644
--- a/kernel/selinux/rules.c
+++ b/kernel/selinux/rules.c
@@ -94,7 +94,22 @@ void apply_kernelsu_rules()
     ksu_allow(db, "system_server", KERNEL_SU_DOMAIN, "process", "getpgid");
     ksu_allow(db, "system_server", KERNEL_SU_DOMAIN, "process", "sigkill");
 
-    mutex_unlock(&ksu_rules);
+#ifdef CONFIG_KSU_SUSFS
+    // Allow umount in zygote process without installing zygisk
+    //ksu_allow(db, "zygote", "labeledfs", "filesystem", "unmount");
+    susfs_set_priv_app_sid();
+    susfs_set_init_sid();
+    susfs_set_ksu_sid();
+    susfs_set_zygote_sid();
+#endif // #ifdef CONFIG_KSU_SUSFS
+
+    rcu_assign_pointer(selinux_state.policy, pol);
+    synchronize_rcu();
+    ksu_destroy_sepolicy(old_pol);
+
+    reset_avc_cache();
+out_unlock:
+    mutex_unlock(&selinux_state.policy_mutex);
 }
 
 #define MAX_SEPOL_LEN 128
